### DISCUSSION TITLE
Training materials that you would like to share

### DISCUSSION THREAD

**Discussion by rowlandm** [RSEAA/RSEAA.github.io#142] [Discussion topics for #RSEAA24]
eg. what training materials or links would you think is of interest to the community?

**Comment by rowlandm** [comment]
[#statstab](https://disabled.social/tags/statstab) #178 PsyTeachR

Thoughts: With term starting soon, here is a collection of free, open, and reproducible resources for teaching psych, methods, and open science.

[#openscience](https://disabled.social/tags/openscience) [#education](https://disabled.social/tags/education) [#psychology](https://disabled.social/tags/psychology) [#eduTwitter](https://disabled.social/tags/eduTwitter) [#r](https://disabled.social/tags/r)

https://psyteachr.github.io/

**Comment by robmoss** [comment]
Shameless self-promotion: [Git is my lab book](https://git-is-my-lab-book.net/) is a set of training materials for EMCRs working in computational infectious diseases epidemiology.

We're currently preparing an [Introduction to Debugging workshop](https://git-is-my-lab-book.net/pr-preview/pr-75/community/training/debugging/) for the imminent SPECTRUM NHMRC CRE annual meeting.

**Update:** the workshop materials have been merged into the main branch and are now [available here](https://git-is-my-lab-book.net/community/training/debugging/)
